src/resmom_win/pbs_open_demux.c

```diff
@@ -120,11 +125,13 @@ main(int argc, char *argv[])
 	(void)strncat_s(pipename_append, _countof(pipename_append), "mom_demux", _TRUNCATE);
 	(void)strncat_s(pipename_append, _countof(pipename_append), this_host, _TRUNCATE);
 	if (create_std_pipes(&si, pipename_append, 0) != 0) {
+		log_err(-1, __func__, "Failed to create pipe");
 		winsock_cleanup();
 		exit(1);
 	}
 	if (connectstdpipes(&si, 0) != 0) {
 		/* close the standard out/err handles before returning */
+		log_err(-1, __func__, "Failed to connect to std pipe");
 		close_valid_handle(si.hStdOutput);
 		close_valid_handle(si.hStdError);
 		winsock_cleanup();
```

Comment:
Should we handle following error condition for 
retval 	-1 	if buffer size (cmd_len), is insufficent.
in line 140: if (0 != get_cmd_shell(cmd_shell, _countof(cmd_shell))). Because function get_cmd_shell in failure case will return -1 with no log messages.

Reply:
It should be handled in util.c changes as part of the Libwin changes I believe.